### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Add a resource via Azure Active Directory_Connection Summary_AI4D : Name "To get missing image description...." is assigned to decorative image.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/AzureStorageExplorer#3510]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG4.1.2;#Win32;#A11ySev2;#Benchmark;#AI4D;#FTP;#WCAG1.3.1;#NVDA;#JAWS;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### **Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with azure AD" screen opened and navigate to Next button and hit enter.
5. Enter the required details in edit boxes and hit enter on "Next" button.
6. "Connection Summary" pop up will appear.
7. Open AI4D and Check the name assigned to graphic label below Tenant label. 

### **Actual:**
Name "To get missing image description...." is assigned to decorative image graphic label below Tenant label. 
Screenreader is reading name for decorative image graphic label below Tenant label.

### **Expected:**
Name should not be assigned to decorative image graphic label below Tenant label. 

### **User Impact:**
Screen reader users will get incorrect information and get confused

### **Attachment**

https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-prraha_microsoft_com/EQFa_5Q-wtpIlVK1XNXcAn0BHNCwXwsnW-ZbeSWmYpMuZA?e=CSP6jH

**Comment by craxal** [comment]
The image in question is content of a `::before` pseudo-element, and screen readers are generating a name for it. Recommendations I'm seeing say we shouldn't use pseudo-elements even for decorative content.

**Comment by Rama8345** [comment]
@craxal We are not able to find the exact control. Could you please help us.

**Comment by Rama8345** [comment]
As per the confirmation on to the mail the product has been changed and this test case does not apply anymore. So, closing the bug. Thanks
**Github Tags:**#Closed;
